packages/integration-tests/tests/me.test.ts

```diff
@@ -0,0 +1,75 @@
+import { ArbitraryObject, User } from '@logto/schemas';
+
+import api, { authedAdminApi } from '@/api';
+import { generatePassword, generateUsername } from '@/utils';
+
+const createUser = async () => {
+  const username = generateUsername();
+
+  return authedAdminApi
+    .post('users', {
+      json: {
+        username,
+        password: generatePassword(),
+        name: username,
+      },
+    })
+    .json<User>();
+};
+
+describe('api `/me`', () => {
+  it('should get user custom data', async () => {
+    const user = await createUser();
+    const customData = await api
+      .get('me/custom-data', {
+        headers: {
+          'development-user-id': user.id,
+        },
+      })
+      .json<ArbitraryObject>();
+
+    expect(customData).toEqual({});
+  });
+
+  it('should update user custom data', async () => {
```

Comment:
nit: maybe better merge these two cases?

Reply:
Just want to keep the test more focused on their test targets.